content/en/blog/_posts/2019-sig-docs-survey.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: blog
+title: "Kubernetes Documentation Survey"
+date: 2019-09-19
+slug: Kubernetes-Documentation-End-User-Survey
+---
+
+**Author:** Aimee Ukasick and SIG Docs
+
+In September, SIG Docs conducted its first survey about the [Kubernetes documentation](https://kubernetes.io/docs/). It was definitely a learning experience. We'd like to thank the CNCF's Kim McMahon for helping us create the survey and access the results.
+
+# Key takeaways
+
+- Respondents would like to see more example code, more detailed content, and more diagrams in the Concepts, Tasks, and Reference sections
+- 74% of respondents would like the Tutorials section to contain advanced content
+- 69.70% said the Kubernetes documentation is the first place they look for information about Kubernetes
+
+# Survey methodology and respondents
+
+The survey was conducted in English and only available for 4 days due to time constraints. We announced the survey on Kubernetes mailing lists, in Kubernetes Slack channels, on Twitter, and in Kube Weekly. There were 23 questions, and respondents took an average of 4 minutes to complete the survey.
+
+Quick facts about respondents:
+
+ - 48.48% are experienced Kubernetes users, 26.26% expert, and 25.25% beginner
+ - 57.58% use Kubernetes in both administrator and developer roles
+ - 64.65% have been using the Kubernetes documentation for more than 12 months
+ - 95.96% read the documentation in English
+
+We asked why people access the Kubernetes documentation, and the majority answered for the Concepts.
+
+{{< figure
+    src="/images/blog/2019-sig-docs-survey/Q9-k8s-docs-use.png"
+    alt="Why respondents access the Kubernetes documentation"
+>}}
+
+This deviated only slightly from what we see in Google Analytics: of the top 10 page hits year to date, #1 was the kubectl cheatsheet in the Reference section, followed overwhelmingly by pages in the Concepts section.
+
+We asked respondents to record their level of satisfaction with the detail in the Concepts, Tasks, Reference, and Tutorials sections.
+
+- Concepts: 47.96% Moderately Satisfied
+- Tasks: 50.54% Moderately Satisfied
+- Reference: 40.86% Very Satisfied
+- Tutorial: 47.25% Moderately Satisfied
+
+We also asked how we could improve each section, providing respondents with selectable answers as well as a text field. The clear majority would like more example code, more detailed content, more diagrams, and advanced tutorials.
+
+- Personally, would like to see more analogies to help further understanding.
```

Comment:
Would it make more sense to fence responses in a code block?

Reply:
done